CoEpi/AppDelegate.swift

```diff
@@ -11,7 +11,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     internal var window: UIWindow?
```

Comment:
per the guide to making ios12 compatible - @i-schuetz does it matter if this is internal var vs just `var window: UIWindow?`?

Reply:
It's internal by default. If you can make it private that'd be ideal but if not, leaving it as just `var window: UIWindow?` should be fine

Reply:
k, thanks. All of the examples of making iOS12 compatible referenced it without it being internal so wasn't sure if our current use would snafu on iOS12. will let @i-schuetz make the call on whether to leave as-is (internal) or not, and confirm with iOS 12 testing later.

Reply:
When I load the debug screen on an iPhone 6s running iOS 12.4, there's no way to slide it down to get back to the home screen, and I have to kill the app to get out of the debug screen. Is this likely related, or something different?

Reply:
I suspect it's unrelated, so I'll open a separate issue.